ImagePicker/Resource/JVTImageFilePicker.h

```diff
@@ -22,6 +22,7 @@
 
 @interface JVTImageFilePicker : NSObject <UIDocumentPickerDelegate, UIDocumentMenuDelegate, JVTImagePreviewVCDelegate>
 @property (nonatomic,assign) CGSize imageResizeSize;
+@property (assign) bool isFilePickerEnabled;
```

Comment:
The vast majority of apps would use this control to just upload a picture, so File upload is not always needed. This property can be set after instantiating the picker in order to hide / show the file picker.